src/pages/index.js

```diff
@@ -6,57 +6,174 @@ import PopupWithImage from "../components/PopupWithImage";
 import Section from "../components/Section";
 import UserInfo from "../components/UserInfo";
 import { initialCards, config } from "../utils/constants";
+import Api from "../utils/Api.js";
+import DeleteConfirmation from "../components/DeleteConfirmation.js";
 
 // DOM Elements
 //tells JavaScript, "This is the spot where we’ll put the cards!"
 const cardsWrap = document.querySelector(".cards__list");
+const editAvatarModalEl = document.querySelector("#avatar__modal");
+const editAvatarForm = editAvatarModalEl.querySelector(".modal__form");
 
 const editProfileModalEl = document.querySelector("#profile__edit-modal");
 const profileEditForm = editProfileModalEl.querySelector(".modal__form");
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "070eb85e-643e-4e8f-9d15-8b589c285fbd",
+    "Content-Type": "application/json",
+  },
+});
+
+let section;
+
+api.getInitialCards().then((cards) => {
+  console.log(cards);
+  section = new Section(
+    {
+      items: cards,
+      renderer: (data) => {
+        section.addItem(renderCard(data));
+      },
+    },
+    ".cards__list"
+    // cardsWrap.prepend('');
+  );
+  section.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**